src/Seq.js

```diff
@@ -192,7 +192,11 @@ export class ArraySeq extends IndexedSeq {
 
 class ObjectSeq extends KeyedSeq {
   constructor(object) {
-    const keys = Object.keys(object);
+    let symbols = [];
+    if (typeof Object.getOwnPropertySymbols === 'function') {
+      symbols = Object.getOwnPropertySymbols(object);
+    }
+    const keys = Object.keys(object).concat(symbols);
```

Comment:
I think we should at least avoid concatenating in the common case where there are no symbols defined in the object.

Reply:
Of course.